mcc/odin/odin/snapshot_list_test.go

```diff
@@ -17,3 +21,39 @@ var exampleSnapshot1 = &rds.DBSnapshot{
 	MasterUsername:       aws.String("owner"),
 	Status:               aws.String("available"),
 }
+
+type listSnapshotsCase struct {
+	testCase
+	name       string
+	snapshots  []*rds.DBSnapshot
+	instanceID string
+}
+
+var listSnapshotsCases = []listSnapshotsCase{
+	// No snapshots for any instance
+	{
+		testCase: testCase{
+			expected:      []*rds.DBSnapshot{},
+			expectedError: "",
+		},
+		name:       "No snapshots for any instance",
+		snapshots:  []*rds.DBSnapshot{},
+		instanceID: "",
+	},
+}
+
+func TestListSnapshots(t *testing.T) {
+	svc := newMockRDSClient()
+	for _, test := range listSnapshotsCases {
+		t.Run(
+			test.name,
+			func(t *testing.T) {
+				actual, err := odin.ListSnapshots(
+					test.instanceID,
+					svc,
+				)
```

Comment:
These last 4 lines are  the *Act*, where the test actually uses the tested code.
In this case, it is invoking the `ListSnapshots` functions with the `instanceID` field of the corresponding `listSnapshotsCase`. Right now this will be `""`, so no instance is to be selected.